### PR TITLE
Fix EGL vendor issue

### PR DESCRIPTION
**Pull request by andyneff**

### CODE REVIEW

**Comment by andyneff** [comment]
The location is inconsistent at best. It' smissing in ubuntu 22.04 install, but it's there in a ubi8 test. I think the best thing for the recipe to do is just put it in both places just in case.

|OS|File|
|--|--|
|Ubuntu 16.04|[/usr/local/share/glvnd/egl_vendor.d/10_nvidia.json](https://gitlab.com/nvidia/container-images/opengl/-/blob/ubuntu16.04/glvnd/runtime/Dockerfile#L50)|
|Ubuntu 18.04|[/usr/share/glvnd/egl_vendor.d/10_nvidia.json](https://gitlab.com/nvidia/container-images/opengl/-/blob/ubuntu18.04/glvnd/runtime/Dockerfile#L14)|
|Ubuntu 20.04|[/usr/share/glvnd/egl_vendor.d/10_nvidia.json](https://gitlab.com/nvidia/container-images/opengl/-/blob/ubuntu20.04/glvnd/runtime/Dockerfile#L14)|
|Centos 7|[/usr/local/share/glvnd/egl_vendor.d/10_nvidia.json](https://gitlab.com/nvidia/container-images/opengl/-/blob/centos7/glvnd/runtime/Dockerfile#L48)|
|Centos 8|[/usr/local/share/glvnd/egl_vendor.d/10_nvidia.json](https://gitlab.com/nvidia/container-images/opengl/-/blob/centos8/glvnd/runtime/Dockerfile#L48)|

File: 40_install_cudagl [hunk 1]
```diff
@@ -142,3 +141,13 @@ elif command -v apt-get 2>&1 > /dev/null; then
...
+
+mkdir -p  /usr/share/glvnd/egl_vendor.d/
+cat - << EOF > /usr/share/glvnd/egl_vendor.d/10_nvidia.json
+{
+    "file_format_version" : "1.0.0",
+    "ICD" : {
+        "library_path" : "libEGL_nvidia.so.0"
+    }
+}
+EOF
```
Comment (andyneff) [inline 40_install_cudagl:153]:
```suggestion
EOF

mkdir -p  /usr/local/share/glvnd/egl_vendor.d/
cp /usr/share/glvnd/egl_vendor.d/10_nvidia.json /usr/local/share/glvnd/egl_vendor.d/10_nvidia.json
```

**Review by scott-vsi — APPROVED** [review]
seems reasonable

**Review by decrispell — APPROVED** [review]
Seemed to work for me!

**Pull request merged**